content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -330,21 +320,17 @@ clusters, zones, or regions.
 This model affords the following benefits:
 
 - Improved availability: If a control plane becomes unavailable, the scope of
-  the outage is limited to only that control plane.
+  the outage is limited to only workloads in clusters managed by that control plane.
 
 - Configuration isolation: You can make configuration changes in one cluster,
   zone, or region without impacting others.
 
-- Controlled Rollout: You have more fine-grained control over configuration
-  rollout (e.g. one cluster at a time). For large meshes with many clusters,
-  this process can be automated with CI/CD pipelines.
+- Controlled rollout: You have more fine-grained control over configuration
+  rollout, for example, to canary configuration changes in a sub-section of the mesh
```

Comment:
The new wording seems less clear to me.

Reply:
I was trying to eliminate duplication, it talked about the same things here and earlier, so I combined them. Maybe it just needs a little more detail now? How about:
```suggestion
  rollout (e.g., one cluster at a time). You can also canary configuration changes in a sub-section of the mesh
```

Reply:
Sure that works.